scripts/PerfHarness/PerfHarness.cs

```diff
@@ -0,0 +1,34 @@
+using System.IO;
+using System.Reflection;
+using Microsoft.Xunit.Performance.Api;
+
+public class PerfHarness
+{
+    public static void Main(string[] args)
+    {
+        using (XunitPerformanceHarness harness = new XunitPerformanceHarness(args))
+        {
+            foreach(var testName in EnumeratePerfTests())
+            {
+                harness.RunBenchmarks(GetTestAssembly(testName));
+            }
+        }
+    }
+
+    private static string GetTestAssembly(string testName)
+    {
+        // Assume test assemblies are colocated/restored next to the PerfHarness.
+        return Path.Combine(
+            Path.GetDirectoryName(Assembly.GetEntryAssembly().Location),
+            $"{testName}.dll"
+        );
+    }
+
+    private static string[] EnumeratePerfTests()
+    {
+        return new [] {
+            "System.Binary.Base64.Tests",
```

Comment:
The list here is actually the list of test assemblies and not tests.  Thus, we won't discover test assemblies, but once we know which are the test assemblies, we'll discover the tests within them (that works today).  Perhaps we should change this to "GetTestAssemblies()".